### ISSUE TITLE
pangram: misleading description about non-ASCII symbols

### ISSUE THREAD

**Issue by Vitorvgc** [exercism/c#262]
The description states:

> The alphabet used consists of ASCII letters a to z, inclusive, and is case insensitive. **Input will not contain non-ASCII symbols.**

Although, the two last test cases involve non-ASCII symbols:

```c
void test_non_ascii_characters(void)
{
   TEST_IGNORE();
   const char sentence[] =
       "Victor jagt zwölf Boxkämpfer quer über den großen Sylter Deich.";

   TEST_ASSERT_TRUE(is_pangram(sentence));
}

void test_pangram_in_alphabet_other_than_ascii(void)
{
   TEST_IGNORE();
   const char sentence[] =
       "Широкая электрификация южных губерний даст мощный толчок подъёму сельского хозяйства.";

   TEST_ASSERT_FALSE(is_pangram(sentence));
}
```

I think that either the description should clarify that input can contain non-ASCII symbols or these test cases should be removed.

**Comment by wolf99** [comment]
I think this is a case where the implementation and tests here have not been updated to the latest version of the problem specification. The current problem specification does not include these inputs. 

Furthermore there may be other exercise that exhibit similar issues due to the same problem. There is ongoing discussion in #261 and there has been discussion in the problem-specifications repo about this, however the issue may be disappearing in nextercism (I am unclear on if an how this is actually happening).

For this particular problem @Vitorvgc, would you be bale to submit a pull request with changes to make the pangram exercise tests match the current problem specification's canonical data? ([https://github.com/exercism/problem-specifications/tree/master/exercises/pangram](https://github.com/exercism/problem-specifications/tree/master/exercises/pangram))

**Comment by Vitorvgc** [comment]
Yes, I can work on it. I'll submit a PR soon.

**Comment by Vitorvgc** [comment]
Fixed by #263
